test/sim/InitializeTracks.test.hh

```diff
@@ -0,0 +1,101 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2020 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file InitializeTracks.test.hh
+//---------------------------------------------------------------------------//
+#include "physics/base/Interaction.hh"
+#include "physics/base/SecondaryAllocatorPointers.hh"
+#include "physics/base/SecondaryAllocatorView.hh"
+#include "physics/base/ParticleParams.hh"
+#include "physics/base/ParticleStateStore.hh"
+#include "physics/base/ParticleTrackView.hh"
+#include "geometry/GeoParamsPointers.hh"
+#include "geometry/GeoStatePointers.hh"
+#include "sim/TrackInitializerStore.hh"
+#include "sim/VacancyStore.hh"
+
+namespace celeritas_test
+{
+using namespace celeritas;
+
+//---------------------------------------------------------------------------//
+// TESTING INTERFACE
+//---------------------------------------------------------------------------//
+//! Interactor
+struct Interactor
+{
+    __device__ Interactor(const ParticleTrackView& particle,
```

Comment:
This is being included from a `.cc` file, so I'm surprised the line compiles -- one of the other headers above must have a CUDA APi include that you're indirectly including. You can put this in a `.cuh`, move it into the `.test.cu` file, or change the decorators to `CELER_FUNCTION`.

Reply:
Yikes, good catch.